lettuce/flows/couette.py

```diff
@@ -11,20 +11,28 @@
 class CouetteFlow2D(object):
     def __init__(self, resolution, reynolds_number, mach_number, lattice):
         self.resolution = resolution
+        self.shape = (resolution, resolution)
+        self._mask = np.zeros(shape=self.shape, dtype=bool)
```

Comment:
you are using self.shape only in self._mask. Perhaps delete this variable ;-)

Reply:
@VascoSch92 sorry for the late reply and thanks for the suggestion. shape is a inherent field of any flow object as it can be important for post-processing. We are planning to integrate more generic classes such as the Flow class, so that such things will be clearer.